tracker/tracker-assist/src/Assist.ts

```diff
@@ -247,6 +261,9 @@ export default class Assist {
       callingAgents.set(id, name)
       updateCallerNames()
     })
+    socket.on('videofeed', ({ streamId, enabled, }) => {
+      videoFeeds[streamId] = enabled
```

Comment:
Why do you need a Proxy-observer with tons of `ts-ignore` lines to maintain reaction behaviour when you could just use a `setVideoFeed()` function?
Use the KISS principal, please.

Besides, it will be better to incapsulate all this logic (except for the socket message part, obviously) inside the CallWindow instance, since it is related to a View only.